### ISSUE TITLE
Beta-Mod is always shown even if there is no recent beta

### ISSUE THREAD

**Issue by duk3luk3** [FAForever/client#480]
Beta-Mod is always shown in featured mod list. It should only be there if there is actually a beta release that's newer than the current fafmod shown as "FA Forever" featured mod.

Not sure if this needs help from server or if we could parse mod_info to figure out if current fafbeta is actually newer than current faf mod.

**Comment by aeoncleanse** [comment]
Hmm.. Given that the beta, just after release, is the same as release... I see no harm. It never remains so for long.

**Comment by duk3luk3** [comment]
Users want more information about what is in beta, otherwise it's just "the buggy version of faf nobody knows anything about" and it won't be played.

**Comment by duk3luk3** [comment]
```
18:05:00 < McNeil> TheDuke: thx. but i don't mean the changelog of the actual patch. i wrote "FAF Beta"...
18:05:14 < TheDuke> is there a new beta?
18:06:03 < McNeil> good question. on the "Play" tab i can start a FAF Beta game...
18:06:20 < TheDuke> that's probably a beta version of newest patch
18:06:33 < TheDuke> because I can't see a new release
18:07:01 < McNeil> if i start a FAF BEta game, the game version is 3661...
```

**Comment by Viba** [comment]
Yes, simple information like: version number, last updated, is it expected to be stable/known crashes, anything specific that needs testing, would help getting beta played more and testing process better.

**Comment by duk3luk3** [comment]
Having a "more info" button next to every featured mod that goes to something on the website, or maybe in the mod vault, having that info would be pretty good.

**Comment by Viba** [comment]
Ideally you'd get something like this when pressing the FAF beta once, or hovering over. [http://puu.sh/sg4Co/fc53398be7.png](http://puu.sh/sg4Co/fc53398be7.png)
Maybe a subtle (i) icon after "FAF Beta" too to indicate extra information.
